Tasks/GooglePlayPromoteV3/task.json

```diff
@@ -116,9 +116,9 @@
             "visibleRule": "rolloutToUserFraction = true"
         },
         {
-            "name": "сleanTheSourceTrack",
+            "name": "cleanTheSourceTrack",
```

Comment:
Will it break already existing pipelines?
If yes, can we notify somehow customers to use right version of the input?

Reply:
I think no, because the variable name with unicode 'c' was same at everywhere. the existing version (3.208.0) works also.